userscript.js

```diff
@@ -162,7 +163,8 @@
     oschina: 'https://www.oschina.net/action/GoToLink?url=',
     weixindev:'https://developers.weixin.qq.com/community/middlepage/href?href=',
     qqdocs: 'https://docs.qq.com/scenario/link.html?url=',
-    pixiv: 'https://www.pixiv.net/jump.php?url='
+    pixiv: 'https://www.pixiv.net/jump.php?url=',
+    anonymz: 'https://anonymz.com/?'
```

Comment:
如果这个网站的外链跳转格式是 `https://anonymz.com/?url=*` ，建议改成这样
```suggestion
    anonymz: 'https://anonymz.com/?url=*'
```
上面的[第 22 行](https://github.com/OldPanda/Open-the-F-king-URL-Right-Now/pull/22/files#diff-5cd4037ed217d72c140b8270f967eab08cd73388ad2b42ed0bb22f011afbdbdfR22)同样